mongo_connector/oplog_manager.py

```diff
@@ -520,6 +523,23 @@ def get_oplog_cursor(self, timestamp=None):
             cursor.add_option(8)
         return cursor
 
+    def _update_config_file(self, excluded_fields):
+        temp_config_location = '/tmp/mongo-connector.json'
+        try:
+            with open(self.config_file, 'r') as configFile:
+                original = json.load(configFile)
+                for field in excluded_fields:
```

Comment:
Could this not just be: original['exclude_fields'] = excluded_fields?
Or, if the excluded_fields in this method doesn't contain all of those in the config, just: original.setdefault('excludedfields', []).extend(excluded_fields).

Either way, I don't think we need a for loop?